src/websocket/stream.py

```diff
@@ -1,24 +1,23 @@
 import asyncio
 import json
 import time
-import uuid
-import websockets
 import traceback
+import uuid
+
+from fastapi import WebSocket, WebSocketDisconnect
 from pydub import AudioSegment
+
 from src.helper import logger
 from src.helper.data_handler import DataHandler
 from src.websocket.stream_transcriber import Transcriber
 
-# To Calculate the seconds of audio in a chunk of 16000 Hz, 2 bytes per sample and 1 channel (as typically used in Whisper):
-# 16000 Hz * 2 bytes * 1 channel = 32000 bytes per second
```

Comment:
Please do keep this in there. I think this is very crucial information otherwise it gets very close to the anti pattern of a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) (I am aware its not quite the same due to its name but still too similar imo)